### PR TITLE
DTSPO-2173  Mirgrate JD public

### PR DESCRIPTION
**Pull request by mokainos**
https://tools.hmcts.net/jira/browse/DTSPO-2173

### CODE REVIEW

File: charts/jd-public/Chart.yaml [hunk 1]
```diff
@@ -0,0 +1,11 @@
+name: jd-public
+apiVersion: v2
+version: 0.1.0
+appVersion: "2.15.129"
+description: Juror Digital - Public
+maintainers:
+  - name: HMCTS Platform Operations
+dependencies:
+  - name: nodejs
+    version: 2.3.5
```
Comment (timja) [inline charts/jd-public/Chart.yaml]:
```suggestion
    version: 2.3.6
```

File: charts/jd-public/Chart.yaml [hunk 2]
```diff
@@ -0,0 +1,11 @@
+name: jd-public
+apiVersion: v2
+version: 0.1.0
+appVersion: "2.15.129"
+description: Juror Digital - Public
+maintainers:
+  - name: HMCTS Platform Operations
```
Comment (timja) [inline charts/jd-public/Chart.yaml]: I would remove this we don't maintain it

File: charts/jd-public/Chart.yaml [hunk 3]
```diff
@@ -0,0 +1,11 @@
+name: jd-public
```
Comment (timja) [inline charts/jd-public/Chart.yaml:1]: a values.yaml file should be created with https://github.com/hmcts/shared-services-flux/blob/master/k8s/stg/releases/juror-digital/jd-public.yaml#L44-L48

File: charts/jd-public/Chart.yaml (hunk 2 above)
Comment (mokainos) [reply charts/jd-public/Chart.yaml]: removed the entry

File: charts/jd-public/Chart.yaml (hunk 3 above)
Comment (mokainos) [reply charts/jd-public/Chart.yaml:1]: added values file with the entries

**Review by timja — APPROVED** [review]

**Pull request merged**